frontend/src/api/k8s/servingRuntimes.ts

```diff
@@ -110,48 +129,46 @@ export const updateServingRuntime = (
     data,
     existingData.metadata.namespace,
     existingData,
+    true,
   );
 
-  const updatedServingRuntime = _.merge(existingData, servingRuntime);
+  // At the moment we won't support editing the serving runtime template
+  const updatedServingRuntime = { ...servingRuntime };
 
   if (!data.tokenAuth) {
-    delete updatedServingRuntime?.metadata?.annotations?.['enable-auth'];
+    delete updatedServingRuntime.metadata?.annotations?.['enable-auth'];
```

Comment:
It's just because we were ommiting the annotation, and since we were carrying all the annotations for the old metadata, we needed to ensure we weren't sticking with that, but checking the code of the controller https://github.com/opendatahub-io/odh-model-controller/blob/7401ea91d7aef225d772ceff9be47055d0c93dae/controllers/inferenceservice_route.go#L206 if we add it with a false value will work, I'm gonna refactor that part.